Extension/package.json

```diff
@@ -2652,6 +2652,16 @@
           "markdownDescription": "%c_cpp.configuration.suggestSnippets.markdownDescription%",
           "scope": "resource"
         },
+        "C_Cpp.caseSensitiveFileSupport": {
+          "type": "string",
+          "enum": [
+            "default",
```

Comment:
Do we have a follow-up task to change the PascalCase enum values to camelCase for 1.13?

Reply:
I'm seeing some potential complications with the rename...

```
  "when": "config.C_Cpp.intelliSenseEngine == 'Default'
```

Reply:
This doesn't affect our feature, does it?

Reply:
It doesn't affect this PR, but we want to make sure we're tracking the work.  Can you open an issue for this and put it in 1.13?

Reply:
Will do!